### ISSUE TITLE
[BUG] On fresh install, exits at the install step with no indication of reason

### ISSUE THREAD

**Issue by supercheetah** [bottlesdevs/Bottles#813]
Please don't fill issue for a Windows executable that doesn't work in Bottles. Our software is a Wine prefix manager. If a Windows application doesn't work, it could be a Wine problem (**Wine is NOT a software developed by Bottles**) or a misconfiguration of the wineprefix (take a look at WineHQ: https://www.winehq.org) or ProtonDB: https://www.protondb.com for more information on the software you are trying to run). 

We will support the installation of Windows software, only through our official installers: https://github.com/bottlesdevs/Bottles/issues/55 (when these are ready).
Please DO NOT open issues for this reason. Issues should be used to report Bugs and request New Features for Bottles.

**Describe the bug**
On a fresh install, and first start, Bottles just exits when it gets to the step to install the runners, and doesn't install any runners.

**Installation**
 - Method: Flatpak
 - Version 2021.12.14-treviso-4

**To Reproduce**
1. Install the Flatpak from Flathub
2. Click 'Next' on the introductory dialogs until it gets to the step about installing the Runners.
3. Click 'Install' for the runners
4. Window disappears, but I captured this from the terminal:
````
❯ flatpak run com.usebottles.bottles
Gtk-Message: 04:48:15.122: Failed to load module "appmenu-gtk-module"
Gtk-Message: 04:48:15.334: Failed to load module "canberra-gtk-module"
(04:48:15) INFO Performing Bottles checks... 
(04:48:16) INFO Bottles Started! 
(04:48:29) INFO Performing Bottles checks... 
(04:48:29) WARNING No dxvk found. 
(04:48:29) INFO Installing component: [dxvk-1.9.2].
````

**Expected behavior**
A window showing that it's installing the runners, whatever that looks like.

**Desktop (please complete the following information):**
 - OS: Pop!_OS
 - Version 21.10
 - Display server: Xorg
**Additional context**

````
❯ flatpak info com.usebottles.bottles

Bottles - Run Windows software

          ID: com.usebottles.bottles
         Ref: app/com.usebottles.bottles/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 2021.12.14-treviso-4
     License: GPL-3.0-or-later
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 51.7 MB
     Runtime: org.gnome.Platform/x86_64/41
         Sdk: org.gnome.Sdk/x86_64/41

      Commit: 7e871de14ec935e654140814602996e31d2ca8ee72301bb8befc40872f7335ad
      Parent: 012f7a713372753bfa0a9e90e0d293a4297028563b6520859914d074f117989f
     Subject: 2021.12.14-treviso-4 (308dffe5)
        Date: 2021-12-18 19:15:05 +0000

````

**Comment by mirkobrombin** [comment]
Hi, this happens sometimes in the Onboard, we are still investigating what causes it

**Comment by SkanderHelali** [comment]
```
flatpak run com.usebottles.bottles
Gtk-Message: 02:58:38.709: Failed to load module "canberra-gtk-module"
(02:58:39) INFO Performing Bottles checks... 
(02:58:39) INFO DXVKs found:
         - dxvk-1.9.2
 
(02:58:39) INFO VKD3Ds found:
         - vkd3d-v2.4
 
(02:58:42) INFO Bottles Started! 

** (bottles:2): WARNING **: 02:58:42.094: AT-SPI: Could not obtain desktop path or name


** (bottles:2): WARNING **: 02:58:42.110: atk-bridge: get_device_events_reply: unknown signature

** (bottles:2): WARNING **: 02:58:42.110: atk-bridge: get_device_events_reply: unknown signature

** (bottles:2): WARNING **: 02:58:42.110: atk-bridge: GetRegisteredEvents returned message with unknown signature

** (bottles:2): WARNING **: 02:58:42.117: AT-SPI: Could not obtain desktop path or name


** (bottles:2): WARNING **: 02:58:42.123: AT-SPI: Could not obtain desktop path or name


** (bottles:2): WARNING **: 02:58:42.123: AT-SPI: Could not obtain desktop path or name

(02:58:45) INFO Performing Bottles checks... 
(02:58:45) INFO DXVKs found:
         - dxvk-1.9.2
 
(02:58:45) INFO VKD3Ds found:
         - vkd3d-v2.4
 
(02:58:45) WARNING No nvapi found. 
(02:58:45) INFO Installing component: [dxvk-nvapi-v0.4]. 
Downloading dxvk-nvapi-v0.4.tar.gz: 100% [==================================================>

(02:58:48) INFO Renaming [dxvk-nvapi-v0.4.tar.gz] to [dxvk-nvapi-v0.4.tar.gz]. 
(02:58:48) INFO NVAPIs found:
         - dxvk-nvapi-v0.4
 
(02:58:48) WARNING No runners found. 
(02:58:49) INFO Installing component: [vaniglia-6.23]. 
Downloading vaniglia-6.23-x86_64.tar.gz: 1% [>
```

Similar issue on OpenSUSE Tumbleweed - silent crash.
Version 2021.12.14-treviso-4.

**Comment by supercheetah** [comment]
I did what @SkanderHelali did and ran it a couple more times, and it did eventually get through the install portion, but I did get the output from one last time that it exited at the Install step with maybe more helpful error messages:
````
❯ flatpak run com.usebottles.bottles
Gtk-Message: 19:01:54.560: Failed to load module "appmenu-gtk-module"
Gtk-Message: 19:01:54.772: Failed to load module "canberra-gtk-module"
(19:01:54) INFO Performing Bottles checks... 
(19:01:55) INFO Bottles Started! 
(19:02:00) INFO Performing Bottles checks... 
(19:02:00) WARNING No dxvk found. 
(19:02:00) INFO Installing component: [dxvk-1.9.2]. 
**
Gtk:ERROR:../gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
Bail out! Gtk:ERROR:../gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
**
Gtk:ERROR:../gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
Bail out! Gtk:ERROR:../gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
````
So I guess the workaround for this bug is to just keep running it through the Install step until it actually goes through.

**Comment by mirkobrombin** [comment]
fixed by 29567a14e2485650fce3df3ef665b96cb7b1d77a
